public/app/main.ts

```diff
@@ -1,6 +1,8 @@
 import {bootstrap}      from 'angular2/platform/browser'
 import {AppComponent}   from './app.component'
+import 'rxjs/Rx';
```

Comment:
The UserService uses rxjs library functions. Rxjs is apparently an async and event-drive library that reduces the cognitive overhead of writing the aforementioned type of javascript. Angular2 explodes if you don't do this import.